src/raiden_libs/utils/signing.py

```diff
@@ -138,21 +137,11 @@ def eth_sign(
         v: int = 27,
         hasher: Hasher = eth_sign_sha3,
 ) -> bytes:
-    warnings.warn(
```

Comment:
Should we remove these warnings? It sounds like we should remove these functions and call functions from `raiden`, instead.

Reply:
I was mostly annoyed by the warning, but I can remove the commit before merging